content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 

```

Comment:
```suggestion
engageons à rendre la participation dans notre communauté 
engageons à faire de la participation à notre communauté
```

Reply:
If, as you said, we want to keep the intent of the English version, this could be a good change !